### PR TITLE
Confirmation before sending image via Cmd/Ctrl+V

### LINKED ISSUE #196: Confirmation for sending image via paste action
Currently Messenger will automatically send any image data in the clipboard upon a paste action (`Ctrl/Cmd+V`), without any confirmation from the user. This can lead to unintentional sending of images.

Other messaging services like Telegram offers a confirmation prompt before sending the image in the clipboard.

We have the ability to [check the format of the clipboard content](https://github.com/electron/electron/blob/master/docs/api/clipboard.md#clipboardavailableformatstype) and alter the default behavior of the paste action.

### PR DESCRIPTION
**Pull request by veniversum**
Fixes #196.

### CODE REVIEW

**Comment by sindresorhus** [comment]
Can you add this to the readme too?

File: index.js [hunk 1]
```diff
@@ -153,6 +154,24 @@ app.on('ready', () => {
 
 	const argv = require('minimist')(process.argv.slice(1));
 
+	electronLocalShortcut.register(mainWindow, 'CmdOrCtrl+V', () => {
+		if (electron.clipboard.availableFormats().some(type => type.includes('image'))) {
+			electron.dialog.showMessageBox({
+				type: 'info',
+				buttons: ['Send', 'Cancel'],
+				message: 'Are you sure you want to send image in clipboard?',
```
Comment (sindresorhus) [inline index.js]: > Are you sure you want to send the image in the clipboard?

**Comment by veniversum** [comment]
Rebased and added edits as requested. If you want to include a screenshot in the readme, I think it would be better if you do it since that'll maintain the look and feel from the other screenshots.

**Comment by sindresorhus** [comment]
I don't think we need a screenshot for this.

**Comment by sindresorhus** [comment]
Thanks :)

You done some good pull requests now. Would you be interested in joining the project? No worries if not.

**Comment by veniversum** [comment]
Yes I would!

**Comment by sindresorhus** [comment]
Welcome! I'm on Gitter if you want to discuss anything in realtime. https://gitter.im/sindresorhus/meta

**Pull request merged**